content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -298,48 +298,37 @@ Configure the Journals
+   :guilabel:`Fiduciary VAT` number and the :guilabel:`Company VAT` number that will be used for
+   the connection. If they are incorrect, they can be changed in
+   :menuselection:`Settings --> Users & Companies --> Companies`.
+#. Click on :guilabel:`Create connection`.
 
-Then, to configure the connection itself, go to
-:menuselection:`Accounting --> Configuration --> Accounting: CodaBox`.
+    * If this is your **first connection**, the wizard will confirm that the connection has been
+      created on **Odoo's side**. Follow the steps to validate the connection on **CodaBox's side**
+      too.
 
-The procedure differs depending on whether **CodaBox** is
-being setup for a first client or an additional one.
+      .. image:: belgium/codabox_connection_validation.png
+        :align: center
+        :alt: Connection validation wizard
 
-.. tabs::
-
-   .. tab:: First connection
-
-       #. Click on :guilabel:`Connect`, where you are redirected to
-          the **CodaBox** platform, here validate the connection.
-       #. Once redirected back to Odoo, the :guilabel:`Status` should be set to
-          :guilabel:`Connected` and a new field :guilabel:`Access Token` should appear.
-
-       .. image:: belgium/codabox_configuration_connection_ok.png
-          :align: center
-          :alt: Configured connection.
-
-       .. note::
-          This :guilabel:`Access Token` will be used to create new connections
-          for other clients without having to validate the connection again.
-
-   .. tab:: Following connection
+      .. note::
+        This :guilabel:`Fiduciary Password` will be used to create new connections
+        for other clients without having to validate the connection again. This password
+        is unique to Odoo and must be stored securely on your side.
 
-       #. Copy the :guilabel:`Access Token` from a previous connection.
-       #. Click on :guilabel:`Connect`.
+    * If this is **not your first connection**, the :guilabel:`Fiduciary password` provided by Odoo
+      during the first connection will be requested to create a new connection.
 
-      The **CodaBox** connection linked to the :guilabel:`Access Token` should still exist,
-      therefore there is no need to validate the connection again. A new and different
-      :guilabel:`Access Token` is generated, and the :guilabel:`Status` should be set to
-      :guilabel:`Connected`.
+      .. image:: belgium/codabox_connection_password.png
+        :align: center
+        :alt: Password request wizard
+#. The :guilabel:`Status` should have now switched to :guilabel:`Connected`.
 
-       .. note::
-          By having a different :guilabel:`Access Token` for each client, you can
-          easily revoke the access of a client to your CodaBox account without
-          affecting the other clients. One client cannot interfere with another.
+    .. image:: belgium/codabox_connection_ok.png
+       :align: center
+       :alt: Connection status OK
```

Comment:
I don't think this is necessary. We tend to avoid having too many pictures for maintenance purposes https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#images